app/routes/datasets/versions.py

```diff
@@ -467,8 +467,9 @@ async def _get_raster_fields(asset: ORMAsset) -> List[RasterBandMetadata]:
     fields: List[RasterBandMetadata] = []
 
     # Magic field which is the area of the region satisfying the query
-    args: Dict[str, Any] = {"pixel_meaning": "area__ha"}
-    fields.append(RasterBandMetadata(**args))
+    for magic in ["area__ha", "latitude", "longitude"]:
```

Comment:
I think I missed the original PR. While I kind of love calling it magic, I feel like something like "reserved_fields" would be more descriptive. But honestly this also makes sense, so whatever you think works best

Reply:
Thanks for the comment!  Yes, changed "magic" to "reserved" in the code and in the documentation.